compiler/pipes/early-optimization.cpp

```diff
@@ -0,0 +1,110 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2022 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include "compiler/pipes/early-optimization.h"
+
+#include "auto/compiler/rewrite-rules/early_opt.h"
+#include "compiler/compiler-core.h"
+#include "compiler/data/function-data.h"
+#include "compiler/function-pass.h"
+#include "compiler/vertex-util.h"
+
+class EarlyOptimization final : public FunctionPassBase {
```

Comment:
Can you please add a comment, what this pass is about?
It's a bit confusing having both auto-generated "early opt pass" and a manually written one.